account_statement_import_txt_xlsx/models/account_statement_import_sheet_mapping.py

```diff
@@ -123,6 +172,18 @@ class AccountStatementImportSheetMapping(models.Model):
     bank_account_column = fields.Char(
         help="Partner's bank account",
     )
+    footer_lines_count = fields.Integer(
+        string="Footer lines number",
+        help="Set the Footer lines number."
+        "Used in some csv file that integrate meta data in"
+        "last lines.",
+        default="0",
+    )
+    column_labels_row = fields.Integer(
```

Comment:
This field name and explanation is not correct. You only read the labels from one column. I think this should be redirect to `header_lines_skip_count` and default to 0.